bolt-sidecar/src/crypto/ecdsa.rs

```diff
@@ -51,19 +54,38 @@ impl ECDSASigner {
     }
 }
 
+/// A generic signing trait to generate ECDSA signatures.
+#[async_trait::async_trait]
+pub trait SignerECDSAAsync: Send + Sync + Debug {
```

Comment:
If we never use the synchronous trait, let's drop that and rename this to `SignerECDSA` (same with `SignerBLS`)